services/web-app/components/page-tabs/page-tabs.module.scss

```
@@ -41,43 +41,60 @@
   }
 }
 
-// override carbon, fix arrow right misalignment
-:global(.cds--tabs .cds--tab--overflow-nav-button--next) {
-  right: convert.rem(-16px);
-  bottom: convert.rem(1px);
+.column {
+  z-index: map.get(variables.$z-indexes, 'floating');
 }
 
-// override Carbon min height and border bottom styles
-.tab-list {
-  min-height: convert.rem(48px);
+.list {
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  padding: 0;
+  list-style: none;
+  scrollbar-width: none;
+}
 
-  :global(.cds--tabs__nav-link) {
-    max-width: fit-content;
-    padding: 0;
-    border-bottom: spacing.$spacing-02 solid transparent;
-    color: theme.$text-secondary;
-    @include type.type-style('body-short-02', true);
+.list-item {
+  display: flex;
 
-    &:hover:not(:focus) {
-      border-bottom: spacing.$spacing-02 solid theme.$layer-active-01;
-      color: theme.$text-primary;
-    }
+  &:not(:first-of-type) {
+    margin-left: 1px;
   }
+}
 
-  // stylelint not recognizing class selectors in sass module global syntax
-  :global(.cds--tabs__nav-item--selected) {
-    border-bottom: spacing.$spacing-02 solid theme.$border-interactive;
-    @include type.type-style('heading-02');
+@media screen and (prefers-reduced-motion: reduce) {
+  .link {
+    transition: none;
   }
 }
 
-.tab-link {
+.link {
+  @include type.type-style('body-compact-02');
+
   display: block;
+
+  height: 3rem;
   padding: spacing.$spacing-04 spacing.$spacing-05 spacing.$spacing-03;
+  border-bottom: spacing.$spacing-02 solid transparent;
   color: theme.$text-secondary;
+  // needed for focus state to be visible
+  outline-offset: -2px;
   text-decoration: none;
+  //transitioning all creates a bug on focus bc of outline transition
+  transition: background motion.$duration-moderate-01 motion.$standard-easing,
+    border-top motion.$duration-moderate-01 motion.$standard-easing;
+
+  white-space: nowrap;
+}
```

```suggestion

  &:focus {
    outline: 2px solid theme.$focus;
    outline-offset: -2px;
  }

  &:hover {
    border-bottom: spacing.$spacing-02 solid theme.$layer-active-01;
  }
}
```

I'm not seeing focus states anymore, and hover seems a little off. Moving hover up here I think you can remove the ` .link:hover:not(:focus)` styles.